.github/ISSUE_TEMPLATE/Resource_proposal.yml

```diff
@@ -0,0 +1,39 @@
+name: New resource proposal
```

Comment:
What "resource" means here?

Reply:
This is a default sampler created issue template. 

We should probably use the word function here and update this to make sense for Fabric Tools.

Reply:
agreed - rewritten...